src/main/java/com/icthh/xm/tmf/ms/activation/web/rest/TenantResource.java

```diff
@@ -21,32 +22,37 @@
 
     @Override
     @PreAuthorize("hasPermission({'tenant':#tenant}, 'ACTIVATION.TENANT.CREATE')")
+    @PrivilegeDescription("Privilege to add a new activation tenant")
```

Comment:
suggest description:
`Privilege to add a new tenant on activation micro-service`